pkgs/applications/audio/espeak-ng/default.nix

```diff
@@ -1,5 +1,6 @@
 { stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkg-config
```

Comment:
```suggestion
{ stdenv, lib, fetchFromGitHub
```

all unused

Reply:
don't forget to squash when merging. github can do that

Reply:
Sorry that was wrong. @jtojnar you have an idea what hammering did here?

mbrola:
<details>


Unused argument: `autoconf`.
Near pkgs/applications/audio/mbrola/default.nix:1:33:

```
  |
1 | { stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkg-config }:
  |                                 ^
```
Unused argument: `automake`.
Near pkgs/applications/audio/mbrola/default.nix:1:43:

```
  |
1 | { stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkg-config }:
  |                                           ^
```
Unused argument: `which`.
Near pkgs/applications/audio/mbrola/default.nix:1:53:

```
  |
1 | { stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkg-config }:
  |                                                     ^
```
Unused argument: `libtool`.
Near pkgs/applications/audio/mbrola/default.nix:1:60:

```
  |
1 | { stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkg-config }:
  |                                                            ^
```
Unused argument: `pkg-config`.
Near pkgs/applications/audio/mbrola/default.nix:1:69:

```
  |
1 | { stdenv, lib, fetchFromGitHub, autoconf, automake, which, libtool, pkg-config }:
  |                                                                     ^
```
</details>

Reply:
That warnings originated in mbrola, which already has been fixed.

Reply:
I need to add upstream the commit link to nixpkgs-review.